cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/utils/streaming/CassandraFile.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cassandra.spark.utils.streaming;
+
+public interface CassandraFile
```

Comment:
Could you please add a short JavaDoc here explaining the purpose of this empty interface?

Reply:
Are there really no common methods? All the usages I can see are in `common` so not sure who would use this interface?

Seems like in practice the usage is always something concrete that extends `CassandraFile` but at that point the concrete class is doing all the heavy lifting.

Reply:
This interface is somewhat in anticipation of Cdc and bringing in a `CommitLog` implementation, when that happens there will be some common  methods.